backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -43,14 +43,45 @@
   }
 }
 
+// Page Layout
+//---------------------------------------------------
+
+html {
+  height: 100%;
+}
+
+body {
+  position: relative;
+  min-height: 100%;
+}
+
+.admin-nav {
+  @include position(fixed, 0 null 0 0);
```

Comment:
Ya, it's just a double-ended affix when the main content is longer than the nav is longer than the viewport.